Havit/Havit/Screens/Main/View/MainTableViewController.swift

```diff
@@ -99,14 +93,51 @@ extension MainTableViewController: UITableViewDataSource {
     }
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return Section.init(rawValue: section)?.numberOfRows ?? 0
+        let section = Section.init(rawValue: section)
+        guard var rowCount = section?.numberOfRows else { return 0 }
+        
+        switch section {
+        case .reach:
+            rowCount = isDeleted ? max(rowCount - 1, 0) : rowCount
+        default:
+            break
+        }
+        
+        return rowCount
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell: ReachRateNotificationTableViewCell = tableView.dequeueReusableCell(
-            withType: ReachRateNotificationTableViewCell.self, for: indexPath)
+        let section = Section.init(rawValue: indexPath.section)
         
-        return cell
+        switch section {
+        case .reach:
+            let row = ReachSection.init(rawValue: indexPath.row)
+            guard var rowValue = row?.rawValue else { return UITableViewCell() }
+            rowValue = isDeleted ? rowValue + 1 : rowValue
+            
+            switch rowValue {
+            case 0:
+                let cell: ReachRateNotificationTableViewCell = tableView.dequeueReusableCell(
+                    withType: ReachRateNotificationTableViewCell.self, for: indexPath)
+                cell.updateNotification(to: "도달률이 50% 이하로 떨어졌어요!")
+                cell.didTapCloseButton = {
+                    self.isDeleted = true
+                    tableView.deleteRows(at: [IndexPath.init(row: ReachSection.notification.rawValue, section: Section.reach.rawValue)], with: .fade)
```

Comment:
p5;
한 줄이 전체적으로 길어질때 저는 이런식으로 parameter 를 enter 를 통해 분리하는게 더 ㄴ눈에 잘 들어오더라구여~
```suggestion
                    tableView.deleteRows(at: [IndexPath.init(row: ReachSection.notification.rawValue,
                                                             section: Section.reach.rawValue)],
                                         with: .fade)
```

Reply:
반영하겠습니다!